t/testrules.yml

```diff
@@ -0,0 +1,11 @@
+---
+# TAP::Harness test rules
+# "t\02-xsdynamic.t" (and possibly "t\03-xsstatic.t") should *not* be run in parallel
```

Comment:
This should not be racy. If it is that's a bug we should fix and not something we should work around.

Reply:
It is racy on Windows.

If you can point to a problem area to search, I'll spend some time looking into it. But it's intermittent (as these things are) which increases the difficulty of tracking it down.
Plus, it may be an artifact of the test harness and Windows, completely unrelated to the module. I don't want to go down a rabbit hole for something that hasn't occurred in actual use of the module or has an actual repeatable failing test. So, to me, serializing the tests seems to be a small price to pay to keep the purposeful tests from interfering with each other.

I can put in a FIXME comment and we can open as issue about it if you'd prefer.

Reply:
Is it racy with MSVC or with GCC? Do you have example reports of failures

That test is easily the slowest in the test suite. It's responsible for 30% of the total test time on my machine, which means that on a 3-4 core system this would double the total testing time.

If windows is racy, it probably shouldn't be running parallel. I don't really like making that a problem for all other platforms.

Reply:
It's Strawberry Perl, so it uses `gcc`.

I ran through my AppVeyor builds and can't find a report with the failure, but it was intermittent.

Maybe the best thing would be to drop the serialization of the tests and wait to see if/when it to reappears and address it at that time.

Reply:
Yeah, I suspect that's best

Reply:
Ok, I removed that commit. I'll hold it in a feature branch for future consideration if it comes back up again. Thanks for the input!